### PR TITLE
chore: [#186811835] replace report an issue bar with beta bar

### PR DESCRIPTION
**Pull request by FarazA22**
### Ticket

https://www.pivotaltracker.com/story/show/186811835

### Approach
Remove report a bug bar, hide beta bar on landing page.

## Code author checklist

- [x] I have performed a self-review of my code
- [x] I have created and/or updated relevant documentation (EX: [Engineering Reference/FAQ](https://docs.google.com/document/d/1X4iWSGmBZdHYZ0jGqkwTR3_yyyqI_TiiJptfc8pi9Po)), if necessary
- [x] I have not used any relative imports
- [x] I have checked for and removed instances of unused config from CMS
- [x] I have pruned any instances of unused code
- [x] I have not added any markdown to labels, titles and button text in config
- [x] If I added/updated any values in `userData` (including `profileData`, `formationData` etc), then I added a new migration file
- [x] If I added any new collections to the CMS config, then I updated the search tool and `cmsCollections.ts`

### CODE REVIEW

**Review by seidior — APPROVED** [review]
Code looks good. Passes all automated tests on my local. Observed behavior is expected on desktop and mobile. Test coverage looks good, and I saw your note in the story about the changes to analytics.

**Review by somabadri — APPROVED** [review]
Good job Faraz! Looks good to me, reviewed it in both desktop and mobile.

File: web/src/components/njwds-layout/PageSkeleton.test.tsx [hunk 1]
```diff
@@ -9,27 +8,27 @@ describe("<PageSkeleton />", () => {
     jest.resetAllMocks();
   });
 
-  it("does not render report an issues bar if onboarding is not completed", () => {
-    useMockBusiness({ onboardingFormProgress: "UNSTARTED" });
-
+  it("does not render the beta bar, legal message, and skip main content", () => {
```
Comment (jphechter) [inline web/src/components/njwds-layout/PageSkeleton.test.tsx:11]: [dust] Maybe specify the location is specific to the landing page in the test name

File: web/src/components/njwds-layout/PageSkeleton.test.tsx [hunk 2]
```diff
@@ -9,27 +8,27 @@ describe("<PageSkeleton />", () => {
...
-    expect(screen.queryByTestId("reportAnIssueBar")).not.toBeInTheDocument();
+    expect(screen.queryByTestId("beta-bar")).not.toBeInTheDocument();
+    expect(screen.queryByTestId("legal-message")).not.toBeInTheDocument();
+    expect(screen.queryByTestId("skip-main-content")).not.toBeInTheDocument();
   });
 
-  it("renders report an issues bar after onboarding is completed", () => {
-    useMockBusiness({ onboardingFormProgress: "COMPLETED" });
-
+  it("renders the beta bar, legal message, and skip main content", () => {
```
Comment (jphechter) [inline web/src/components/njwds-layout/PageSkeleton.test.tsx:23]: [dust] Specify the location in the test name

File: web/src/components/njwds-layout/PageSkeleton.tsx [hunk 3]
```diff
@@ -14,19 +12,16 @@ interface Props {
...
         {!props.landingPage && <SkipToMainContent />}
         <Banner />
         <OutageAlertBar />
-        <BetaBar />
       </section>
       <div className="fit-screen-content">{props.children}</div>
       <footer>
-        {business?.onboardingFormProgress === "COMPLETED" && <ReportAnIssueBar />}
+        {!props.landingPage && <BetaBar />}
         {!props.landingPage && <LegalMessage />}
```
Comment (jphechter) [inline web/src/components/njwds-layout/PageSkeleton.tsx:25]: [sand] Why not consolidate these given they're controlled by the same param?

**Pull request merged**